docs/release-notes/mongoid-7.5.txt

```diff
@@ -119,6 +119,61 @@ Embedded associations (``embeds_one`` and ``embeds_many``):
 +---------------------------------------+---------------------------------------+
 
 
+``any_of`` Adds Multiple Arguments As Top-Level Conditions
+----------------------------------------------------------
+
+**Breaking change:** When ``any_of`` is invoked with multiple conditions, the
```

Comment:
Did we conclude that we're going to release this breaking change in 7.5? I thought it had to be in the next major version?

Reply:
Yes, we will redo the versions but for now since everything is referencing 7.5 it would be confusing to have a mix of 7.5 and 8.0 references thus I'm continuing with the existing numbers.

Reply:
It's a good call to put this in 8.0.